app/src/test/java/io/apicurio/registry/rules/validity/XsdContentValidatorTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.rules.validity;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.apicurio.registry.AbstractRegistryTestBase;
+import io.apicurio.registry.content.ContentHandle;
+
+/**
+ * @author cfoskin@redhat.com
+ */
+public class XsdContentValidatorTest extends AbstractRegistryTestBase {
+    @Test
+    public void testValidSyntax() throws Exception {
+        ContentHandle content = resourceToContentHandle("xml-schema-valid.xsd");
+        XsdContentValidator validator = new XsdContentValidator();
+        validator.validate(ValidityLevel.SYNTAX_ONLY, content);
```

Comment:
Same note here as in the WSDL content validator test.  You can add another test within this method to **also** run the SYNTAX_ONLY validity against the `xml-schema-invalid-semantics.xsd` test file and it should be valid.

Reply:
👍